pythonfmu/fmi2slave.py

```diff
@@ -263,3 +268,24 @@ def _fmu_state_to_bytes(state: Dict[str, Any]) -> bytes:
     @staticmethod
     def _fmu_state_from_bytes(state: bytes) -> Dict[str, Any]:
         return json.loads(state.decode("utf-8"))
+
+    def log(self, msg: str, status: Fmi2Status = Fmi2Status.ok, category: str = "", debug: bool = False):
+        """Log a message to the FMU logger.
+        
+        Args:
+            msg (str) : Log message
+            status (Fmi2Status) : Optional, message status (default ok)
+            category (str) : Optional, message category (default "")
+            debug (bool) : Optional, is this a debug message (default False)
+        """
+        if self.logger is not None and self.resources is not None:
+            if self.__lib is None:
+                library_path = Path(self.resources).parent / "binaries" / get_platform() / (self.modelName + "." + get_lib_extension())
+                self.__lib = cdll.LoadLibrary(library_path)
```

Comment:
This fails on Windows. Not expecting a `Path`. Any reason for changing this? Issue on linux?

Reply:
I had to specify the full path on Linux otherwise it was not finding the library.

This is unfortunate that the behavior of `LoadLibrary` is inconsistent between Windows and Linux.